### DISCUSSION TITLE
I need help with the screen firmware.

### DISCUSSION THREAD

**Discussion by Tigrex-Dev** [mriscoc/Ender3V2S1#1325] [Q&A]
Information about my 3D printer
- Ender 3 v2
- Sprite Pro Extruder
- CR-Touch
- 4.2.2 Board
- DWIN Display

Hello, I think the professional firmware is really great, but unfortunately I can't find the source code for the display firmware anywhere. Or can I simply use the standard DWIN Ender 3 v2 display source code from Marlin for the display? I would like to make some changes there.

**Comment by Koky05** [comment]
Did you try look here?
[Giadej compilation](https://github.com/mriscoc/Ender3V2S1/tree/Ender3V2S1-Released/display%20assets/Giadej%20compilation)

**Reply by Tigrex-Dev** [reply]
Yes, I have that, but it is not the source code, so I can't really change much.

**Reply by Tigrex-Dev** [reply]
I'm looking for the source code ^^ because I also want to modify the info panel.
![Bild_2024-07-10_110637555](https://github.com/mriscoc/Ender3V2S1/assets/100427020/f300dc03-6cdf-48be-acc7-54d2dff11902)

**Reply by Koky05** [reply]
If you do not need to change size or visual, information in info panel are located in Marlin FW (Version.h).

**Reply by Tigrex-Dev** [reply]
Hmm thats not right the "Version: " is in the marlin/lib/proui/stm32f1/ directory, in the file named "libproui_ubl_dwin.a" and without the src code i cant change the sign.
